docs/reporting-component-authentication.adoc

```diff
@@ -0,0 +1,47 @@
+
+:imagesdir: images
+:source-highlighter: pygments
+
+ifdef::env-github[]
+// If on GitHub, define attributes so we can find our diagram files and render
+// them.
+
+// The branch will be used to find the correct diagrams to render below.
+// When PRing changes to the diagrams you can change this attributes
+// temporarily to the name of the branch you're working on. But don't forget
+// to change it back to main before merging!!
+:github-branch: spike/MAV-1406-auth-sharing-with-reporting
```

Comment:
Do we still need this if we merge this in to `next`?

Reply:
I've updated it to `main` as that's where it will eventually end up, and it's what all the other adocs have

Reply:
Looking at this block a bit further, I'm confused how it's working because it refers to the repo as `nhsuk/record-childrens-vaccinations` whereas it should be `nhsuk/manage-vaccinations-in-schools`.

Reply:
Hah! I copy-and-pasted this block from one of the other adocs (architecture.adoc IIRC), and looking at those other adocs, they still have that repo reference too (e.g. https://github.com/nhsuk/manage-vaccinations-in-schools/blob/next/docs/architecture.adoc?plain=1#L14 )

...so is that `:github-repo:` line even needed?

Reply:
Yeah, perhaps we can remove that block?